docs/release-notes.md

```diff
@@ -3,6 +3,28 @@
 - Table of Contents
 {:toc}
 
+## Version 1.5.0
+
+The 1.5.0 release of MCUboot adds support for encrypted images using
+ECIES with secp256r1 as an Elliptic Curve alternative to RSA-OAEP. A
+new swap method was added which allows for upgrades without using a
+scratch partition. There are also lots of bug fixes, extra simulator
+testing coverage and some imgtool updates.
+
+### About this release
+
+- TLVs were updated to use 16-bit lengths (from previous 8).
```

Comment:
I wonder if we should mention here that this will break compatibility on big-endian targets?

Reply:
in known issues, yes

Reply:
I added it to the text and under "known issues".